data/questions_gen_postgres.csv

```diff
@@ -33,7 +33,7 @@ What month were most students admitted?,"SELECT date_trunc('month', s.admit_term
 What's the average predicted time to graduation since admission in no. of days?,SELECT avg(predicted_graduation_semester - admit_term) AS average_predicted_time_to_graduation FROM student;,advising,date_functions,
 How many students were predicted to graduate in the last 10 years?,"SELECT count(*) AS num_students_graduated FROM student WHERE predicted_graduation_semester >= DATE_TRUNC('year', CURRENT_DATE) - interval '10 year';",advising,date_functions,
 How long has it been in days since the last admitted student?,SELECT CURRENT_DATE - max(admit_term) AS duration_since_last_admitted_student FROM student;,advising,date_functions,
-Subtract 2 weeks from the most recent predicted graduation date and give the month.,"SELECT extract(MONTH FROM date_trunc('month', predicted_graduation_semester) - interval '2 weeks') AS MONTH FROM student ORDER BY predicted_graduation_semester DESC LIMIT 1;",advising,date_functions,
+Subtract 2 weeks from the most recent predicted graduation date and give the month.,"SELECT DATE_TRUNC('month', s.predicted_graduation_semester - INTERVAL '2 weeks') AS MONTH FROM student s ORDER BY s.predicted_graduation_semester DESC LIMIT 1;SELECT extract(MONTH FROM date_trunc('month', predicted_graduation_semester) - interval '2 weeks') AS MONTH FROM student ORDER BY predicted_graduation_semester DESC LIMIT 1;",advising,date_functions,

```

Comment:
Allow the returning of the truncated date. Previous correct query only returned the month number. Technically both are acceptable since we didn't mention which format we wanted in the question.
```
advising=# SELECT DATE_TRUNC('month', s.predicted_graduation_semester - INTERVAL '2 weeks') AS MONTH FROM student s ORDER BY s.predicted_graduation_semester DESC LIMIT 1;SELECT extract(MONTH FROM date_trunc('month', predicted_graduation_semester) - interval '2 weeks') AS MONTH FROM student ORDER BY predicted_graduation_semester DESC LIMIT 1;
        month        
---------------------
 2022-04-01 00:00:00
(1 row)

 month 
-------
     4
(1 row)
```